cmd/osio/cmd/root.go

```diff
@@ -21,7 +23,7 @@ func init() {
 		Short: "osio is a helper tool for OpenShift.io.",
 	}
 
-	RootCmd.PersistentFlags().StringVarP(&targetEnv, "target", "t", "stage", "Target environment OpenShift.io stage vs prod.")
+	RootCmd.PersistentFlags().StringVarP(&targetEnv, "target", "t", "stage", "target environment OpenShift.io stage vs prod")
```

Comment:
Why would you change this? This is not an error message. Did the linter complain about that? This is the Viper help.